godtossing/Pos/Ssc/GodTossing/Toss/Base.hs

```diff
@@ -461,6 +461,10 @@ checkSharesPayload epoch shares = do
 -- For certificates we check that
 --   * certificate hasn't been sent already
 --   * certificate is generated by richman
+--   * there is no existing certificate with the same VSS key
+--   * the map doesn't contain certificates with the same VSS key (this
```

Comment:
thb it's unclear which certificates you talking about, it would be better to write explicitly.